var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -27,8 +31,9 @@ class ModelHippocampus(SimModel):
     variant('coreneuron',  default=False, description="Enable CoreNEURON Support")
     variant('profile',     default=False, description="Enable profiling using Tau")
 
-    depends_on('neuron+mpi', type=('build', 'link', 'run'))
-    depends_on('coreneuron', when='+coreneuron', type=('build', 'link', 'run'))
+    # We dont link automatically to neuron/corenrn, nrnivmodl does it for us (=> no 'link' mode)
+    depends_on('neuron+mpi', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('neuron+mpi')
```
even we dont link explicitly, they are just hidden inside script...so more clear to keep that?

Reply:
I added a comment to explain well why we dont set link mode.